src/Model/QuickPayPaymentOperation.php

```diff
@@ -8,9 +8,31 @@ class QuickPayPaymentOperation extends QuickPayModel
 {
     public const TYPE_AUTHORIZE = 'authorize';
     public const TYPE_CAPTURE = 'capture';
+    public const TYPE_REFUND = 'refund';
+    public const TYPE_CANCEL = 'cancel';
 
     public const STATUS_CODE_APPROVED = 20000;
 
+    /**
+     * @var int
+     */
+    protected $id;
+
+    /**
+     * @var string
+     */
+    protected $type;
+
+    /**
+     * @var int
+     */
+    protected $amount;
+
+    /**
+     * @var string
+     */
+    protected $qp_status_code;
```

Comment:
```suggestion
    protected $quickpayStatusCode;
```

I would probably call it that